src/lang/en-US.json

```diff
@@ -187,20 +187,20 @@
   "game_mode_6": "Intro",
   "game_mode_7": "Diretide",
   "game_mode_8": "Reverse Captains Mode",
-  "game_mode_9": "Greeviling",
+  "game_mode_9": "The Greeviling",
   "game_mode_10": "Tutorial",
   "game_mode_11": "Mid Only",
   "game_mode_12": "Least Played",
   "game_mode_13": "Limited Heroes",
-  "game_mode_14": "Compendium Matchmaking",
+  "game_mode_14": "Compendium",
   "game_mode_15": "Custom",
   "game_mode_16": "Captains Draft",
   "game_mode_17": "Balanced Draft",
   "game_mode_18": "Ability Draft",
   "game_mode_19": "Event",
-  "game_mode_20": "All Random Death Match",
-  "game_mode_21": "1v1 Mid",
-  "game_mode_22": "All Draft",
+  "game_mode_20": "All Random Deathmatch",
+  "game_mode_21": "1v1 Solo Mid",
+  "game_mode_22": "Ranked All Pick",
```

Comment:
I think we should keep this as All Draft, since even unranked all pick matches use this mode now.